solidity/package.json

```
@@ -18,9 +18,14 @@
     "test:quick": "mocha --exit --recursive --timeout 45000",
     "test:quick:watch": "mocha --watch --exit --recursive --timeout 45000",
     "demo": "truffle migrate --reset && truffle exec ./scripts/delegate-tokens.js",
-    "lint": "solium --dir ./contracts",
+    "lint": "npm run lint:js && npm run lint:sol",
     "lint:js": "eslint ${npm_package_config_eslintPaths}",
-    "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}"
+    "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}",
+    "lint:sol": "solium -d contracts/ && prettier --list-different contracts/ **/*.sol",
```

Meybe we could use [`--check` instead `--list-different`](https://prettier.io/docs/en/cli.html#--check) so we have human readable output? We may need to bump up the version of prettier for that.
```suggestion
    "lint:sol": "solium -d contracts/ && prettier --check contracts/ **/*.sol",
```

---

Changed in 37696c0 but I don't think it helped much. Here's the diff:

--check
```
Checking formatting...
[warn] contracts/libraries/staking/TopUps.sol
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @keep-network/keep-core@1.7.0-pre.0 lint:sol: `solium -d contracts/ && prettier --check '**/*.sol'`
npm ERR! Exit status 1
```

--list-different
```
contracts/libraries/staking/TopUps.sol
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @keep-network/keep-core@1.7.0-pre.0 lint:sol: `solium -d contracts/ && prettier --list-different  '**/*.sol'`
npm ERR! Exit status 1
```

---

Maybe it depends on a file type, but for some types `--check` will print a neat output.